components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -199,64 +209,93 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
             savePreferences(currentUserId, abPreferences);
             setFirstChannelName('');
         }
+        const {onNextNavigateTo, onPrevNavigateTo} = this.props;
+        if (onNextNavigateTo && nextStep === true && autoTour) {
+            onNextNavigateTo();
+        } else if (onPrevNavigateTo && nextStep === false && autoTour) {
+            onPrevNavigateTo();
+        }
+    }
+
+    public handlePrev = (e?: React.MouseEvent): void => {
+        e?.preventDefault();
+        this.handleSavePreferences(true, false);
+    }
+
+    public handleNext = (auto = true, e?: React.MouseEvent): void => {
```

Comment:
same here .. and please make defaulted parameters the last in the param list

Reply:
this is optional because sometimes we are automatically moving next in that case we don’t have a mouse event.

Reply:
mmh, ok ... thats fair ... in that case we can leave the optional with default value in first place